webpack.config.js

```diff
@@ -67,8 +68,16 @@ const config = {
             patterns: [
                 './node_modules/vscode-azureextensionui/resources/**/*.svg',
                 './node_modules/vscode-codicons/dist/codicon.{css,ttf}',
+                './node_modules/open/xdg-open*',
```

Comment:
This is a loose parity change with what's done in vscode-azureextensiondev, but stripped down to only what we need.

Reply:
Maybe add a little more verbiage about what it's doing (or why it's required)?